### DISCUSSION TITLE
Ability to set output path

### DISCUSSION THREAD

**Discussion by AdaskoTheBeAsT** [runtime-env/import-meta-env#4] [Q&A] [answered]
could you please add ability to setup output path which will be checked?
right now it is predefined set of folders
[import-meta-env]: Output file not found: dist/**/*, .next/**/*, .nuxt/**/*, .output/**/*, build/**/*

**Comment by soc221b** [comment, marked as answer]
Hi @AdaskoTheBeAsT 

You can use the `--output` cli argument: https://github.com/iendeavor/import-meta-env/blob/b9461b1112065f44692b3fc4b5016a59616ae722/packages/cli/src/create-command.ts#L27

You can use `--help` to see other available arguments. 😃

**Reply by AdaskoTheBeAsT** [reply]
Thanks :)

**Reply by AdaskoTheBeAsT** [reply]
one more question related to this - as on prod files usually has cache busting part would it be possible to use glob in output?

Edited: checked source code  - indeed there is. 👍 cool
